rita_client/src/dashboard/interfaces.rs

```diff
@@ -161,13 +165,14 @@ fn set_interface_mode(iface_name: &str, mode: InterfaceMode) -> Result<(), RitaC
         return Err(RitaClientError::InterfaceModeError(
             "Attempted to configure non-existant or unavailable interface!".to_string(),
         ));
-    } else if target_mode == InterfaceMode::Wan {
+    } else if target_mode == InterfaceMode::Wan || target_mode == InterfaceMode::LTE {
         // we can only have one WAN interface, check for others
-        // StaticWAN entires are not identified seperately but if they ever are
+        // StaticWAN entries are not identified seperately but if they ever are
         // you'll have to handle them here
         for entry in interfaces {
+            let iface = entry.0;
             let mode = entry.1;
-            if mode == InterfaceMode::Wan {
+            if iface != iface_name && (mode == InterfaceMode::Wan || mode == InterfaceMode::LTE) {
```

Comment:
This avoids an error when changing from WAN to LTE

Reply:
adding that as a comment could be valuable.